action.yml

```diff
@@ -15,7 +15,7 @@ inputs:
   scannerVersion:
     description: Version of the Sonar Scanner CLI to use
     required: false
-    default: 6.2.1.4610
+    default: 6.2.1.4610 # to be kept in sync with sonar-scanner-version
```

Comment:
The default for `scannerVersion` here is redundant with the value in `sonar-scanner-version`, which is used in the two sub-actions (`install-build-wrapper` and in `deprecated-c-cpp`).

For the time being, I have added a comment here. I will explore ways to factorize it in a later PR.